src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -616,6 +617,16 @@ public class SimpleEvents {
 				"\t\tset repair cost to repair cost * 50%",
 				"\t\tsend \"You're LUCKY! You got 50% discount.\" to player")
 			.since("INSERT VERSION");
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerTradeEvent")) {
+			Skript.registerEvent("Player Trade", SimpleEvent.class, PlayerTradeEvent.class, "player trad(e|ing)")
+				.description("Called when a player has traded with a villager.")
+				.requiredPlugins("Paper 1.16.5+")
+				.examples("on player trade:",
+					"\tchance of 50%:",
+					"\t\tcancel event",
+					"\t\tsend \"The trade was somehow denied!\" to player")
+				.since("2.6.3");
```

Comment:
When adding a new feature the `@Since` annotation should be set to `INSERT VERSION` so that when ever it's merged that value will be manually changed to the proper Skript version it was added in.
```suggestion
				.since("INSERT VERSION");
```

Reply:
Thanks! I'll keep this in mind, I knew I forgot something. ;)